app.rb

```diff
@@ -34,7 +34,8 @@ def create_person
       print 'Has parent permission? [Y/N]: '
       permission = gets[0].capitalize
       permission = permission == 'Y'
-      @people.push(Student.new(age, name, parent_permission: permission))
+      # @people.push(Student.new(age, name, parent_permission: permission))
```

Comment:
- [ ] Kindly remove the commented code, in order to make the codebase clean and more readable.